ci/create_local_database.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+import os
+import tempfile
+from typing import List
+
+import typer
+import yaml
+from create_database import create_migration_tables, migrate
+
+from gear import Database
+
+
+def main(service: str, database_name: str):
+    asyncio.run(async_main(service, database_name))
+
+
+async def async_main(service: str, database_name: str):
+    migrations = read_migrations_from_build_yaml(service)
+
+    db = Database()
+    await db.async_init()
+    await db.just_execute(f'CREATE DATABASE IF NOT EXISTS `{database_name}`;')
+
+    os.environ['HAIL_SQL_DATABASE'] = database_name
+    os.environ['HAIL_SCOPE'] = 'dev'
+    os.environ['HAIL_NAMESPACE'] = 'local'
```

Comment:
Can you explain why you have to modify these environment variables after creating the database instead of before?

Reply:
`'HAIL_SQL_DATABASE'` cannot be set at first because the database in the server might not exist. We then need to recreate the `Database` object to pick up that change in configuration once the database *does* exist. It's an unfortunate aspect of the `Database` API as it currently stands.

The rest of the environment variables are used by the migrations so we don't need them for the earlier lines where we're just creating the MySQL database. I can move them up, but just thought to keep all the environment variable setting together.

Reply:
Where is `HAIL_SQL_DATABASE` in the code base? I couldn't find it at all.

Reply:
It's new, I added it to pick up what database to use in the `local_insecure_config` because `local_insecure_config` doesn't have a config on the filesystem.

Reply:
ah, I get it now!